surveys/2021-annual-survey/questions.md

```diff
@@ -676,18 +679,15 @@ Select one:
 
 ### Which best describes your course or activity?
 
-- A programming course which only teaches Rust
-- A programming course which teaches Rust amongst other languages
-- An computer science course (e.g., operating systems, algorithms, etc.) course which uses Rust with or without other languages
-- Other course which uses Rust
-- Individual project
-- Group project
-- Other activity
+- A course teaching exclusively how to program in Rust
+- A course teaching how to program in Rust and other languages
+- A computer science course (e.g., operating systems, algorithms, etc.) course which uses Rust (and potentially other languages)
+- Other type of course where Rust was used
```

Comment:
I think this change is sub-optimal - we are asking the reader to decide whether a course is a computer science course or not which seems a difficult call. We've also lost the distinction between course and project. I would prefer removing 'computer science' from the third option and the forth option to 'a project or similar activity'

Reply:
Can you give me an example of a "project"? All the projects I participated in in university were part of a course. Perhaps there's a distinction between "lecture" and "lab work" that you make that I classify together as a "course".